src/app/app-routing.module.ts

```diff
@@ -1,10 +1,23 @@
 import {NgModule} from '@angular/core';
 import {Routes, RouterModule} from '@angular/router';
 import {AuthenticationGuard} from 'microsoft-adal-angular6';
+import {LandingPageComponent} from '@app/landing-page/landing-page.component';
+import {TechnicalRecordCreateComponent} from '@app/technical-record-create/technical-record-create.component';
 
 const routes: Routes = [
   {path: '', redirectTo: 'home', pathMatch: 'full', canActivate: [AuthenticationGuard]},
-  {path: 'home', loadChildren: './shell/shell.page.module#ShellPageModule' , canActivate: [AuthenticationGuard] },
+  {path: 'home', pathMatch: 'full', component: LandingPageComponent , canActivate: [AuthenticationGuard]},
+  {
+    path: 'home/search',
```

Comment:
`/home`, `/search` and `/create` shall all be independent routes like the technical-record one